### PR TITLE
For Issue #981 Create CustomThemes.md

### PR DESCRIPTION
**Pull request by fiumad**
Outlines a simple way to change the background color of the memos app.

### CODE REVIEW

File: doc/CustomThemes.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Adding A Custom Theme
+
+1. Open the Settings Dialog
+2. Navigate to the System Tab
+3. In the "Additional Styles" box add these lines of code:
+
+`.memo-list-container {background-color: #INSERT COLOR HERE;}`
+
+`.page-container {background-color: #INSERT COLOR HERE;}`
```
Comment (boojack) [inline doc/CustomThemes.md:9]:
```suggestion
\`\`\`css
.memo-list-container {background-color: #INSERT COLOR HERE;}`
.page-container {background-color: #INSERT COLOR HERE;}
\`\`\`
```

File: doc/CustomThemes.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- It is recommended that you choose the same color for both options
+5. Refresh the page and the background color of your memos app will successfully update to reflect your changes
```
Comment (boojack) [inline doc/CustomThemes.md:12]:
```suggestion
  It is recommended that you choose the same color for both options

4. Refresh the page and the background color of your memos app will successfully update to reflect your changes
```

File: doc/CustomThemes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Adding A Custom Theme
+
+1. Open the Settings Dialog
+2. Navigate to the System Tab
+3. In the "Additional Styles" box add these lines of code:
+
+\`\`\`css
```
Comment (boojack) [inline doc/CustomThemes.md:7]:
```suggestion
```css
```

File: doc/CustomThemes.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+.memo-list-container {background-color: #INSERT COLOR HERE;}`
+.page-container {background-color: #INSERT COLOR HERE;}
+\`\`\`
```
Comment (boojack) [inline doc/CustomThemes.md:10]: use "```" instead

**Comment by fiumad** [comment]
Sorry about that, not sure why it added the slashes.

**Comment by boojack** [comment]
> Sorry about that, not sure why it added the slashes.

It's the codeblock syntax in markdown

**Review by boojack — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**